app/models/labware.rb

```diff
@@ -29,7 +29,16 @@ class Labware < Asset
   has_many :requests_as_target, through: :receptacles
   has_many :transfer_requests_as_source, through: :receptacles
   has_many :transfer_requests_as_target, through: :receptacles
+
+  # Submissions in progress for the labware. Found by looking at those associated
+  # with the transfer requests into the receptacles. Be a little cautious using this,
+  # as it will not handle cross-submission pools, but is better for historical
+  # data.
   has_many :submissions, through: :receptacles
+
+  # Direct submissions are those made on the plate itself, and are found via
+  # the orders associated with the well.s
+  has_many :direct_submissions, -> { distinct }, through: :receptacles
```

Comment:
Sorry so many questions on this file...
It looks from the comment like this should be going via a different route through the data model, but looks identical to the above relationship apart from the `distinct` - how does that make it go through wells / orders rather than transfer requests?

Reply:
Ah I see, it uses the new `direct_submissions` method on `receptacle` 👍